extensions/big-data-cluster/src/bigDataCluster/auth.ts

```diff
@@ -0,0 +1,35 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as kerberos from 'kerberos';
+import * as vscode from 'vscode';
+
+export async function authenticateKerberos(hostname: string): Promise<string> {
+	const service = 'HTTP' + (process.platform === 'win32' ? '/' : '@') + hostname;
```

Comment:
No HTTPS support?

Reply:
This is just how authentication protocol looks, it actually does use HTTPS as far as I know. However if I try to change here to "HTTPS" I get the following back:
```
Error during authentication - Miscellaneous failure (see text): Server (HTTPS/mssql.azdata.local@AZDATA.LOCAL) unknown
```